packages/labs/context/src/lib/value-notifier.ts

```diff
@@ -31,12 +31,9 @@ export class ValueNotifier<T> {
   }
 
   public setValue(v: T, force = false) {
-    let changed = false;
-    if (!Object.is(v, this._value)) {
-      changed = true;
-    }
     this._value = v;
-    if (changed || force) {
+
+    if(force || !Object.is(v, this._value)) {
```

Comment:
`!Object.is(v, this._value)` will always return `true` here (except when `v` is `NaN`) because of the previous line.

I think the mostly simplification we could get is:
```ts
    const update = force || !Object.is(v, this._value);
    this._value = v;
    if (update) {
```